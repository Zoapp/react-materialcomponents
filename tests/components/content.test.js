import React from "react";
import renderer from "react-test-renderer";
import Content from "@libs/components/content";

describe("components/Content", () => {
  it("can render children", () => {
    const component = renderer.create(
    	<Content>
    		Children are here!
    	</Content>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("can be fixed", () => {
    const component = renderer.create(
    	<Content 
    	  fixed
      />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
