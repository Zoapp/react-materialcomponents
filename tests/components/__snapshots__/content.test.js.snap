// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Content can be fixed 1`] = `
<div
  className="rmdc-content rmdc-content-fixed-adjust"
/>
`;

exports[`components/Content can render children 1`] = `
<div
  className="rmdc-content rmdc-content-height"
>
  Children are here!
</div>
`;
